src/test/resolver.test.ts

```diff
@@ -1,6 +1,16 @@
 import { Resolver } from "..";
 
 jest.setTimeout(60 * 1000);
+test('Unstoppable Domain resolves to an address', async () => {
```

Comment:
Another case testing the branches of the `try...catch` would be valuable as well. Unless you decide to refactor of course

Reply:
no longer using try catch.